src/omics/README.omics.md

```diff
@@ -19,3 +19,23 @@ List of tools:
 4. **Manifest Log** - Download and parse the manifest log of an Omics run. The manifest log contains a lot of information about storage usage, CPU and memory usage per task, Docker images, inputs, and general information about the run.
 
     Run `uv run manifest_log --help` for more details.
+
+## Run with Docker
+A fast and easy way to run the above-mentioned tools is by running a container:
+
+```sh
+docker run -v ~/.aws:/root/.aws -e AWS_PROFILE=${AWS_PROFILE} -v <local_output>:<output> 337532070941.dkr.ecr.us-east-1.amazonaws.com/ugbio_omics:1.3.4-0dev-54 <tool>
+```
+
+Breakdown:
+
+1. A simple `docker run` command.
+2. `-v ~/.aws:/root/.aws -e AWS_PROFILE=${AWS_PROFILE}` - this will allow the Docker container to use the SSH profile you are logged into (customer/dev/prod).
+    > Remember to use **alog** before running the container.
+3. `-v <local_output>:<output>` - add a volume mapping and use it in the output path of the tool to get the outputs available outside the container.
+4. `337532070941.dkr.ecr.us-east-1.amazonaws.com/ugbio_omics:1.3.4-0dev-54` - Docker image.
+5. `<tool>` - available tools are:
```

Comment:
instead of repeating the tools - add link to the tools above - or put the "Run with Docker" section before and the tools list under

Reply:
I know it is a duplication, but users don't know the tools entry point. The idea is to make it as simple as possible. Above you can find the tool entry point with 'uv run' and it is confusing.

Reply:
ok fair enough